src/Crdt/SyncHelper.cs

```diff
@@ -0,0 +1,77 @@
+using System.Text.Json;
+using Crdt.Core;
+
+namespace Crdt;
+
+public static class SyncHelper
```

Comment:
Seems all the methods are internal
```suggestion
internal static class SyncHelper
```